lib/iex/lib/iex/info.ex

```diff
@@ -3,12 +3,23 @@ defprotocol IEx.Info do
 
   @spec info(term) :: [{atom, String.t}]
   def info(term)
+
+  Kernel.def __protocols__(data) do
```

Comment:
Instead of defining this here and reusing it everywhere, shouldn't we automatically add "Implemented protocols" to all types here: https://github.com/whatyouhide/elixir/blob/ae9d78b98434c5cb7d96ee95aa38f99496021587/lib/iex/lib/iex/helpers.ex#L432